cmd/all.go

```diff
@@ -74,37 +79,66 @@ func startMetricsServer() {
 }
 
 func processRepositories(repositories chan crawler.Repository, processedCounter prometheus.Counter) {
-	ch := make(chan string, 100)
-
-	//throttle requests
-	rate := time.Second // 1 per second. TODO check rate limit (https://confluence.atlassian.com/bitbucket/rate-limits-668173227.html)
+	channelCapacity := 100
+	ch := make(chan string, channelCapacity)
+	counter := 0
```

Comment:
counter is just used to fmt.Println on line 97 (which is commented), can we remove it?

Reply:
Of course. Removed.